Src/Directory.Build.props

```
@@ -22,6 +22,7 @@
     <PackageIcon>sentry-nuget.png</PackageIcon>
     <PackageProjectUrl>https://sentry.io</PackageProjectUrl>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageReadmeFile>README.md</PackageReadmeFile>
```

I have a feeling you need an additional change in order to include the readme file.

```suggestion
    <PackageReadmeFile>README.md</PackageReadmeFile>
    
    <None Include="$(MSBuildThisFileDirectory)../README.md" Pack="true" PackagePath=""/>
```

---

Thanks - was just looking at that! 👍